test/src/test/kotlin/graphql/nadel/tests/EngineTests.kt

```diff
@@ -180,7 +180,11 @@ private suspend fun execute(
 
                                 if (indexOfCall != null) {
                                     val serviceCall = serviceCalls.removeAt(indexOfCall)
-                                    serviceCall.response
+                                    if (serviceCall.incrementalResponse != null) {
+                                        serviceCall.incrementalResponse.initialResponse //for now, just return initial response
+                                    } else {
+                                        serviceCall.response!!
+                                    }
```

Comment:
I think your next PR will cover this, but, ideally, the engine would fail with a descriptive message when fixtures haven't defined neither `incrementalResponse` or `response`.